pkg/cmd/roachtest/operations/backup_restore.go

```diff
@@ -69,12 +69,25 @@ outer:
 		return nil
 	}
 
-	backupTS := timeutil.Now().Add(-1 * time.Minute).UTC().Format(time.DateTime)
-
-	o.Status(fmt.Sprintf("backing up table district in db %s", dbName))
+	o.Status(fmt.Sprintf("backing up table district in db %s (full)", dbName))
 	bucket := fmt.Sprintf("gs://%s/operation-backup-restore/%d/?AUTH=implicit", testutils.BackupTestingBucket(), timeutil.Now().UnixNano())
 
-	_, err = conn.ExecContext(ctx, fmt.Sprintf("BACKUP TABLE %s.district TO '%s' AS OF SYSTEM TIME '%s'", dbName, bucket, backupTS))
+	_, err = conn.ExecContext(ctx, fmt.Sprintf("BACKUP TABLE %s.district INTO '%s' AS OF SYSTEM TIME '-10s'", dbName, bucket))
```

Comment:
nit: perhaps it would be easier to conduct a backup and restore of a database, so we don't need to worry about foreign keys?